### DISCUSSION TITLE
How to update the placement of the popover after it is rendered

### DISCUSSION THREAD

**Discussion by upender7270** [adobe/react-spectrum#7426] [Q&A]
I Have use case where I displayed a popover with content as a menu and on click menu option update the popover content to different UI. This works fine but when the popover trigger is near the bottom of the screen and menu opens with `placement: bottom left` when popover content is updated height is more than the content, here how to update the placement to top if content not fits 

![Screenshot from 2024-11-22 14-22-38](https://github.com/user-attachments/assets/40e16192-cda1-471c-aedd-71ce3cb178ff)
![image](https://github.com/user-attachments/assets/60dd145e-52f7-4a12-a038-a36710d60c83)

**Comment by LFDanLu** [comment]
Mind sharing how you are building this experience (hooks, React Aria Components, React Spectrum)? [useOverlayPosition](https://github.com/adobe/react-spectrum/blob/b0f15697245de74ebc99ab3d687f5eb3733d3a34/packages/%40react-aria/overlays/src/useOverlayPosition.ts#L298) does expose `updatePosition` but that is only at the hook level unfortunately. Alternatively, could you instead just open a modal when triggering the editing?
